### PR TITLE
Assorted tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
@MohammadYounes here are the patches.

Please review without whitespace and see if something is wrong because there were conflicts while cherry picking these :)

### CODE REVIEW

File: lib/rtlcss.js [hunk 1]
```diff
@@ -54,27 +54,26 @@ module.exports = (options, plugins, hooks) => {
 
   return {
     postcssPlugin: 'rtlcss',
-    Once (root, { result }) {
```
Comment (XhmikosR) [inline lib/rtlcss.js:57]: There are more unused function parameters in other places BTW

File: lib/util.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ module.exports = {
       restorer
     }
     state.value = state.value.replace(what, (c) => {
-      if (exclude && c.match(exclude)) {
+      if (exclude && exclude.test(c)) {
```
Comment (XhmikosR) [inline lib/util.js:131]: This needs careful review; if the global flag is used, then this is not the same as before.

File: lib/util.js [hunk 3]
```diff
@@ -211,7 +209,6 @@ module.exports = {
     return this.regexCache[name]
   },
   regex (what, options) {
-    what = what || []
```
Comment (XhmikosR) [inline lib/util.js:214]: Removed the fallback, but happy to add it back if you think it's needed

**Comment by XhmikosR** [comment]
BTW @MohammadYounes I don't see coveralls added to the repo, are you sure you added it properly?

File: lib/util.js (hunk 2 above)
Comment (MohammadYounes) [reply lib/util.js:131]: Yes, global flag is used.

File: lib/util.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ const PATTERN_TOKEN = CHAR_TOKEN_START + '\\d+:\\d+' + CHAR_TOKEN_END // «offse
 const PATTERN_TOKEN_WITH_NAME = '\\w*?' + CHAR_TOKEN_START + '\\d+:\\d+' + CHAR_TOKEN_END // «offset:index»
 
 const REGEX_COMMENT = /\/\*[^]*?\*\//igm // non-greedy
-const REGEX_DIRECTIVE = /\/\*\s*(?:!)?\s*rtl:[^]*?\*\//img
+const REGEX_DIRECTIVE = /\/\*\s*!?\s*rtl:[^]*?\*\//img
```
Comment (MohammadYounes) [inline lib/util.js:20]: Why remove the Non-capturing group ?

**Comment by MohammadYounes** [comment]
> BTW @MohammadYounes I don't see coveralls added to the repo, are you sure you added it properly?

Is there anything to add from repo side ?

**Pull request merged**